Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml

```diff
@@ -1,28 +1,28 @@
-# - type: vessel
-  # id: Anchor
-  # name: KC Anchor
-  # description: A large luxury cruiser capable of long ranged travel acrossed the sector, expedition capable.
-  # price: 135020
-  # category: Large
-  # group: Expedition
-  # shuttlePath: /Maps/_NF/Shuttles/Expedition/anchor.yml
+- type: vessel
+  id: Anchor
+  name: KC Anchor
+  description: A large luxury cruiser capable of long ranged travel acrossed the sector, expedition capable.
+  price: 110100
```

Comment:
Make sure that the markup is 30%

Reply:
For what? No one is going to buy it at a high price anymore.

Reply:
Increased price by 7k for 30% markup. Grid sells for slightly less than 90k ingame.